xchange-coindeal/src/main/java/org/knowm/xchange/coindeal/service/CoindealMarketDataService.java

```diff
@@ -0,0 +1,41 @@
+package org.knowm.xchange.coindeal.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.coindeal.CoindealAdapters;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.marketdata.Ticker;
+import org.knowm.xchange.dto.marketdata.Trades;
+import org.knowm.xchange.service.marketdata.MarketDataService;
+import org.knowm.xchange.service.marketdata.params.Params;
+
+import java.io.IOException;
+import java.util.List;
+
+public class CoindealMarketDataService extends CoindealMarketDataServiceRaw
+        implements MarketDataService {
+
+    public CoindealMarketDataService(Exchange exchange) {
+        super(exchange);
+    }
+
+    @Override
+    public Ticker getTicker(CurrencyPair currencyPair, Object... args) throws IOException {
+        return null;
```

Comment:
this is not a correct behavior for a missing feature
just leave the default implementation in such a case

however getTicker is such a basic functionality I think we should implement it
maybe I'll have time to help in the weekend

Reply:
I have convert it to throw a NotImplementedException